src/it/java/servlets/admin/config/DisableScoreboardIT.java

```diff
@@ -1,258 +1,258 @@
-			}
-			else
-			{
-				//Add Cookies from Response to outgoing request
-				request.setCookies(response.getCookies());
-				String responseBody = doThePost(csrfToken);
-				if(responseBody.contains("Scoreboard is now disabled and cannot be accessed"))
-				{
-					log.debug("Scoreboard was Disabled in message");
-					if(ScoreboardStatus.isScoreboardEnabled())
-					{
-						String message = "Scoreboard was not actually disabled";
-						log.fatal(message);
-						fail(message);
-					}
-				} else {
-					String message = "Admin was unable to disable Scoreboard";
-					log.fatal(message);
-					fail(message);
-				}
-			}
-		}
-		catch (Exception e)
-		{
-			log.fatal("Could not Complete: " + e.toString());
-			fail("Exception Caught: " + e.toString());
-		}
-	}
-
-	/**
-	 * Test with Bad CSRF Tokens
-	 */
-	@Test
-	public void testCsrf()
-	{
-		String userName = "configAdminTester";
-		String password = userName;
-		//Verify / Create user in DB
-		try
-		{
-			TestProperties.verifyTestAdmin(log, applicationRoot, userName, password);
-			//Sign in as Admin User
-			log.debug("Signing in as User Through LoginServlet");
-			TestProperties.loginDoPost(log, request, response, userName, userName, null, lang);
-			log.debug("Login Servlet Complete, Getting CSRF Token");
-			if(response.getCookie("token") == null)
-				fail("No CSRF Token Was Returned from Login Servlet");
-			String csrfToken = response.getCookie("token").getValue();
-			if(csrfToken.isEmpty())
-			{
-				String message = new String("No CSRF token returned from Login Servlet");
-				log.fatal(message);
-				fail(message);
-			}
-			else
-			{
-				//Add Cookies from Response to outgoing request
-				request.setCookies(response.getCookies());
-				String responseBody = doThePost("wrongCSRFToken");
-				if(responseBody.contains("Scoreboard Configuration Failure"))
-				{
-					log.debug("Scoreboard Config Failure Recieved");
-					if(!ScoreboardStatus.isScoreboardEnabled())
-					{
-						String message = "Scoreboard was disabled on what should have been a failed request";
-						log.fatal(message);
-						fail(message);
-					}
-				} else {
-					String message = "No CSRF Error Detected";
-					log.fatal(message);
-					fail(message);
-				}
-			}
-		}
-		catch (Exception e)
-		{
-			log.fatal("Could not Complete: " + e.toString());
-			fail("Exception Caught: " + e.toString());
-		}
-	}
-}
+package servlets.admin.config;
+
+import static org.junit.Assert.*;
+
+import org.apache.log4j.Logger;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.mock.web.MockServletConfig;
+
+import testUtils.TestProperties;
+import utils.InstallationException;
+import utils.ScoreboardStatus;
+
+public class DisableScoreboardIT
+{
+	private static org.apache.log4j.Logger log = Logger.getLogger(DisableScoreboard.class);
```

Comment:
DisableScoreboard should also be renamed to DisableScoreboardIT (this was a mistake in the test too)